package.json

```diff
@@ -167,5 +168,18 @@
   },
   "engines": {
     "node": ">=12.22.0"
+  },
+  "turbo": {
+    "pipeline": {
+      "build-native": {
+        "dependsOn": [
+          "^build-native"
+        ],
+        "outputs": [
+          "native/*.node",
+          "build/swc/target/**"
```

Comment:
Are these being cached together? The target directory was using multiple restore keys because we could reuse parts of it which would speed up the actual build if there was no cache hit.

Reply:
@padmaia I see fixing now.